alto-pathmanager/api/src/main/java/org/opendaylight/alto/ext/helper/PathManagerHelper.java

```diff
@@ -35,6 +40,13 @@
   public static FlowDesc toAltoFlowDesc(Match match) {
     FlowDescBuilder builder = new FlowDescBuilder();
     if (match != null) {
+      if(match.getEthernetMatch()!=null) {
```

Comment:
Keep exact one space between `if` and `(`, **expression** and **operator** (like `!=`).

Please go through your code and check all about this issue.